example/spec/serial/DiscoverGranulesHttpSpec.js

```diff
@@ -102,34 +103,60 @@ describe('The Discover Granules workflow with http Protocol', () => {
         await waitForApiStatus(
           getGranule,
           {
-            prefix: config.stackName,
+            prefix,
             granuleId: granule.granuleId,
           },
           'completed'
         );
         await deleteGranule({
-          prefix: config.stackName,
+          prefix,
           granuleId: granule.granuleId,
         });
       }
     ));
     // Order matters. Child executions must be deleted before parents.
-    await deleteExecution({ prefix: config.stackName, executionArn: ingestGranuleWorkflowArn1 });
-    await deleteExecution({ prefix: config.stackName, executionArn: ingestGranuleWorkflowArn2 });
-    await deleteExecution({ prefix: config.stackName, executionArn: ingestGranuleWorkflowArn3 });
-    await deleteExecution({ prefix: config.stackName, executionArn: discoverGranulesExecutionArn });
-
-    await Promise.all(noFilesIngestExecutionArns.map((executionArn) => deleteExecution({ prefix: config.stackName, executionArn })));
-    await Promise.all(partialFilesIngestExecutionArns.map((executionArn) => deleteExecution({ prefix: config.stackName, executionArn })));
-    await Promise.all(ignoringFilesIngestExecutionArns.map((executionArn) => deleteExecution({ prefix: config.stackName, executionArn })));
+    await Promise.allSettled(
+      runningWorkflows.map((executionArn) =>
+        deleteExecution({
+          prefix,
+          executionArn,
+        }))
+    );
+
+    await deleteExecution({
+      prefix,
+      executionArn: discoverGranulesExecutionArn,
+    });
 
-    await deleteExecution({ prefix: config.stackName, executionArn: ignoringFilesConfigExecutionArn });
-    await deleteExecution({ prefix: config.stackName, executionArn: partialFilesConfigExecutionArn });
-    await deleteExecution({ prefix: config.stackName, executionArn: noFilesConfigExecutionArn });
+    await Promise.all(
```

Comment:
Take/leave re: formatting  I did a block opinionated format via prettier, and generally prefer narrower line lengths, however that might be much for some.